todoapp/projects/serializers.py

```
@@ -0,0 +1,29 @@
+from rest_framework import serializers
+
+from projects import models as projects_models
+from users import serializers as users_serializers
```

Ruff (0.12.2)

`users.serializers` imported but unused

Remove unused import: `users.serializers`

(F401)